libcnb-package/src/cross_compile.rs

```diff
@@ -7,95 +8,71 @@ use which::which;
 /// look for the required tools and returns a human-readable help text if they can't be found or
 /// any other issue has been detected.
 pub fn cross_compile_assistance(target_triple: impl AsRef<str>) -> CrossCompileAssistance {
-    // Background: https://omarkhawaja.com/cross-compiling-rust-from-macos-to-linux/
-    if target_triple.as_ref() == X86_64_UNKNOWN_LINUX_MUSL && cfg!(target_os = "macos") {
-        // There is more than just one binary name here since we also support the binary name for
-        // an older version cross_compile_assistance which suggested installing a different
-        // toolchain.
-        let possible_gcc_binary_names = ["x86_64-unknown-linux-musl-gcc", "x86_64-linux-musl-gcc"];
+    let target = target_triple.as_ref();
+    let (gcc_path, help_text) = match (target, consts::OS, consts::ARCH) {
+        (AARCH64_UNKNOWN_LINUX_MUSL, OS_LINUX, ARCH_X86_64) => (
+            "aarch64-linux-gnu-gcc",
+            "To install an aarch64 cross-compiler on Ubuntu:\nsudo apt-get install g++-aarch64-linux-gnu",
+        ),
+        (AARCH64_UNKNOWN_LINUX_MUSL, OS_MACOS, _) => (
```

Comment:
Since the `_` will match any value, wouldn't it be better to put `ARCH_X86_64 | ARCH_AARCH64` there to be explicit and to ensure we'll end up in the catch-all for unknown architectures?

(There is another `_` match later as well)

Reply:
For sure -- addressed in https://github.com/heroku/libcnb.rs/pull/769/commits/cab120b23e7965ef932119fbe813df3ce74e31f5